### DISCUSSION TITLE
setParagraph <- what is going on here?

### DISCUSSION THREAD

**Discussion by jloganolson** [ueberdosis/tiptap#1706] [General Questions] [answered]
I'm looking at the code for paragraph and see a toggle node with 'paragraph' as both parameters
```
setParagraph: () => ({ commands }) => {
    return commands.toggleNode('paragraph', 'paragraph')
},
```
What is this doing? Why wouldn't you use setNode? Asking to better understand the node extensions (and for future devs in the same boat).

**Comment by philippkuehn** [comment, marked as answer]
Haha good catch! I think this is the result of some copy-pasting 😅 
`setNode` should do the trick.
